pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -134,7 +134,7 @@ def test_get_data_multiple_symbols(self):
     @pytest.mark.parametrize('adj_pr', [True, False])
     def test_get_data_null_as_missing_data(self, adj_pr):
         # workaround as skip_on_exception decorator changes signature
-        # should it be changed to a signature-preserving decorator?
+        # TODO: should it be changed to a signature-preserving decorator?
```

Comment:
~~~python
def test_..
  # TODO: We can't decorate the test function along with
  # parametrization because it errors.  Investigate this later.
~~~